lmdeploy/turbomind/turbomind.py

```diff
@@ -217,6 +218,15 @@ def _from_hf(self, model_source: ModelSource, model_path: str,
         if osp.exists(osp.join(model_path, 'outputs_stats.pth')) and \
                 engine_config.model_format is None:
             engine_config.model_format = 'awq'
+        # update model_format if config.json contains awq quantization config
```

Comment:
After the merge of https://github.com/InternLM/lmdeploy/pull/1799, perhaps these lines could be removed.

Reply:
Yes, they will be removed.